app/processors/base_processor.rb

```diff
@@ -256,7 +256,7 @@ def download_file(uri)
 
   def file_schemes
     @_file_schemes ||= begin
-      s = ['s3', 'ia', 'ftp', 'http']
+      s = ['s3', 'ia', 'ftp', 'http', 'https']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.